pkg/www/retext/src/components/TextRenderer.vue

```diff
@@ -142,21 +142,30 @@
                     currentWord = this.words(indices[0], indices[1])[indices[2]];
                 }
 
-
                 let div = createDiv(e.clientX, e.clientY);
                 div.innerText = "\"" +  selectedWords.reduce((acc, val) => {return acc + " " + val;}) + "\"";
 
                 document.body.appendChild(div);
 
                 let move = (e) => {
-                    div.style.left = e.clientX + "px";
-                    div.style.top = e.clientY + "px";
+                    div.style.left = (1 + e.clientX) + "px";
+                    div.style.top = (e.clientY) + "px";
                 }
 
                 let remove = () => {
                     div.remove()
                     document.removeEventListener("mouseup", remove);
                     document.removeEventListener("mousemove", move);
+
+                    let words = JSON.parse(JSON.stringify(this.dragTool));
+                    words.documentID = this.$parent.selected;
+                    words.text = selectedWords.join(" ");
+                    // this.$emit("dragStop", words);
+
+                    this.channel.two(words, () => {
+                        // sample callback
+                        console.log(this.$parent.selected, words)
```

Comment:
So I sort've understand that you want to clean up after yourself. But I don't think that's strictly necessary. I suppose the bad case is something like 

1) words get picked up

2) the user drops them somewhere (not on a drop area)

3) words are now still in channel, but user is no longer dragging anyting

4) later use hits a mouseup event over drop area, without actually dragging anything.

5) we're in trouble.

I would say though based on the way events bubble in javascript you can probably always safely "purge" the channel by the time the event bubbles up to the document. This comment is more my thoughts, and perhaps you've already figured this out.